.coveragerc

```diff
@@ -31,4 +31,4 @@ relative_files = true
 source =
   tests
 source_pkgs =
-  awx_plugins.credentials.x.api
+  awx_plugins.credentials
```

Comment:
Should we reference each module file instead? i.e. `awx_plugins.credentials.aim`

Reply:
Maybe. I'm not sure. I haven't tested with multiple modules. But if this works, I'd leave it this way.

I'd even use
```suggestion
  awx_plugins
```
if that's enough to make it function.

Beware that Codecov's v4 action messes up some file paths, though. So don't rely on it, only on the local coverage.py which is integrated through `pytest-cov`.

And with that, I realized I forgot to integrate the HTML report generation into tox :man_facepalming:
Will do that later. But for now, running `tox -- --cov-report=html` should do the trick.